src/types/DocumentTypes.ts

```diff
@@ -198,3 +198,7 @@ export type TemplateDocumentProperties = DocumentProperties;
 export interface ProjectDocumentProperties extends DocumentProperties {
     templateId: string;
 }
+
+export type DocumentLoadOptions = {
+    keepConnectors: boolean;
```

Comment:
Should we add the docs describing each option in here too?

Reply:
@brapoprod WDYT Brian?

It's not done for the other ones and it's still available in `load` doc atm.

Reply:
I indeed think it makes most sense on top of the load function